core/src/it/scala/org/ensime/fixture/SearchServiceFixture.scala

```diff
@@ -12,7 +12,7 @@ import org.ensime.AkkaBackCompat
 import org.ensime.indexer.SearchService
 import org.ensime.vfs._
 
-trait IsolatedSearchServiceFixture extends IsolatedSourceResolverFixture with AkkaBackCompat {
+trait IsolatedSearchServiceFixture extends IsolatedSourceResolverFixture with AkkaBackCompat with EnsimeConfigFixture {
```

Comment:
this doesn't look right, this fixture already has an `EnsimeConfig` injected... all you need to do is put

```
import EnsimeConfigFixture.serverConfig
```

in the imports of any file that creates a `SearchService`